src/tools/std23/views/chunk.hpp

```diff
@@ -70,9 +70,10 @@ struct chunk_view : std20::ranges::view_interface< chunk_view< R > > {
     using iterator_category = std::conditional_t<
                                   std20::ranges::random_access_range< Base >,
                                   std20::random_access_iterator_tag,
-                                  std::conditional_t< std20::ranges::bidirectional_range< Base >,
-                                                  std20::bidirectional_iterator_tag,
-                                                  std20::forward_iterator_tag > >;
+                                  std::conditional_t<
+                                      std20::ranges::bidirectional_range< Base >,
+                                      std20::bidirectional_iterator_tag,
+                                      std20::forward_iterator_tag > >;
 
```

Comment:
This is just a style change, yes? Is the goal to be consistent in this style? Can I put it like this:
if items inside < > extend more than 1 entry, then use a line break and add one entry per new line?

If this is the objective for consistent styling, template on line 80 needs to also be adjusted

Reply:
I did this mainly for readability (the conditional is like a ternary statement). The style of these standard-like pieces of code is not as fixed as I would like it. I basically have not decided if we should use our normal style or something more like the nanorange style.